uberfire-nio2-backport/uberfire-nio2-impls/uberfire-nio2-jgit/src/main/java/org/uberfire/java/nio/fs/jgit/JGitFileSystemProvider.java

```diff
@@ -1793,6 +1794,11 @@ public void setAttribute( final Path path,
         checkNotNull( "path", path );
         checkNotEmpty( "attributes", attribute );
 
+        if ( attribute.equals( SquashOption.SQUASH ) && value instanceof SquashOption ) {
```

Comment:
just to follow codestyle, would you mind change SquashOption.SQUASH to SquashOption.SQUASH_ATTR ?